setup.py

```diff
@@ -65,7 +66,7 @@
         "phylo-treetime >=0.11.2, <0.12",
         "pyfastx >=1.0.0, <3.0",
         "scipy ==1.*",
-        "xopen[zstd] >=1.7.0, ==1.*"
+        "xopen[zstd] >=1.7.0, <3"
```

Comment:
I did a quick spot check of the [CI jobs](https://github.com/nextstrain/augur/actions/runs/9898268001) and only found environments that resolved to version 2.0.2. This means we're not testing on version 1.x anymore.

One option is to add xopen to the matrix like we do with biopython, but I think it's overkill.

I'll run the tests locally where I have version 1.8.0. As long as they pass it should be fine.

Reply:
xopen 1.9.0 is resolved in the pathogen-repo-ci environments and those passed. Specifically `CI / pathogen-repo-ci (ncov)` [uses](https://github.com/nextstrain/augur/actions/runs/9898268001/job/27344675938?pr=1532#step:8:141) a `.xz` file as input to augur filter, and that was successful, so we should be good here.

Reply:
I've added something to deprecated.md. It's unlikely to break given the import works and we're testing with conda (until we've adjusted the recipe). Thanks for checking!